### PR TITLE
Add documentation for the 'elasticsearch-translog' tool

### PR DESCRIPTION
**Pull request by dakrone**
This adds documentation to the translog page for the CLI truncation
tool.

### CODE REVIEW

File: docs/reference/index-modules/translog.asciidoc [hunk 1]
```diff
@@ -70,4 +70,57 @@ update, or bulk request.  This setting accepts the following parameters:
...
     automatic commit will be discarded.
---
\ No newline at end of file
+--
+
+[float]
+[[corrupt-translog-truncation]]
+=== What to do if the translog becomes corrupted?
+
+In some cases (a bad drive, user error) the translog can become corrupted, when
```
Comment (abeyad) [inline docs/reference/index-modules/translog.asciidoc]: corrupted, when -> corrupted. When

File: docs/reference/index-modules/translog.asciidoc [hunk 2]
```diff
@@ -70,4 +70,57 @@ update, or bulk request.  This setting accepts the following parameters:
...
+to the node, recovering from a replica if available.
+
+If there is no copy of the data from which Elasticsearch can recover
+successfully, a user may want to recover the data that is part of the shard at
+the cost of losing the data that is currently contained in the translog. We
+provide a command-line tool for this, `elasticsearch-translog`.
+
+[WARNING]
+The `elasticsearch-translog` tool should *not* be run while Elasticsearch is
+running, and you will permanently lose the documents that contained only in the
```
Comment (abeyad) [inline docs/reference/index-modules/translog.asciidoc]: that were contained

**Comment by abeyad** [comment]
LGTM, couple minor comments, no need for another round

**Comment by dakrone** [comment]
Thanks Ali!

**Pull request merged**